bip-0039.mediawiki

```diff
@@ -6,7 +6,7 @@
           Pavol Rusnak <stick@satoshilabs.com>
           Aaron Voisine <voisine@gmail.com>
           Sean Bowe <ewillbefull@gmail.com>
-  Comments-Summary: Unanimously Discourage for implementation
+  Comments-Summary: The creation of word lists moved to a separate repo: https://github.com/p2w34/wlips
```

Comment:
This doesn't belong here.

If you wish to leave a comment, put it at the Comments-URI wiki page. (But that's for review of this BIP, not changes proposed to it.)

Reply:
I see. I revert the change to this line.